src/Spice86.Core/Emulator/Devices/Video/Registers/Enums/CrtControllerRegister.cs

```diff
@@ -0,0 +1,29 @@
+namespace Spice86.Core.Emulator.Devices.Video.Registers.Enums;
+
+public enum CrtControllerRegister {
```

Comment:
Missing documentation that explains what each value means.

Reply:
It's just the names of the registers. But I added docs to shut the analyzer up ;)